google-cloud-bigtable/lib/google/cloud/bigtable/flat_row.rb

```diff
@@ -35,19 +64,26 @@ def == other # :nodoc:
 
         attr_accessor :key, :cells
 
+        # Create flat row object
+        #
+        # @param key [String] Row key name
+
         def initialize key = nil
           @key = key
           @cells = Hash.new { |h, k| h[k] = [] }
         end
 
-        # Column families
-        # @return [String]
+        # List of column families
+        # @return [Array<String>]
 
         def column_families
           @cells.keys
         end
 
-        def == other # :nodoc:
+        # FlatRow object comparator
```

Comment:
This method was `:nodoc:` before, should it be `@private` now?

Reply:
Added private